user_docs/OpenSARlab_environment.md

```diff
@@ -0,0 +1,137 @@
+# OpenSARlab Environment
+Every OpenSARlab user has access to an Amazon AWS EC2 instance. Individual instances are shared
+among groups of 1 - 3 users, depending on demand.
+
+### Operating System
+- Ubuntu 18.04
+
+### Volume (storage)
+OpenSARlab uses Amazon AWS EBS volume for user storage. After a day without use, a snapshot
+of a volume is taken and the volume is destroyed. Upon the next login, a new EBS volume is
+created from the snapshot. This can take some time (tens of minutes) and users may notice that
+notebooks load slowly during this period. The important takeaway is that user storage is 
+persistent; you will not lose saved work.
+
+It is incumbent upon users to manage their storage. Using up all your storage space will 
+result in the inability to login to OpenSARlab. Please contact an OpenSARlab administrator
+if this occurs.
+- 500GB Amazon AWS EBS volume
+
+### Memory (RAM)
+As previously mentioned, EC2 instances are shared among users. This happens behind the scenes and is
+generally not noticeable when using OpenSARlab, with the exception of memory availability. The amount 
+of memory available to each user fluctuates with overall use on an instance, and may vary from 6GB to 16GB.
+- 6GB • 16GB 
+
+### Privileges
+- Users do not have root privileges (no sudo)
+
+### Installed Software
```

Comment:
Many of listed software (like `lib-`) are dependencies and so probably do not need to be shown. 

Others like `alien` are used during the image build to convert the ISCE rpm to something more Ubuntu friendly like deb. It's not expected that users will use it though I guess they could. 

I'm not sure of what software should be publicly known,

Reply:
I wasn't sure about this either, so I just included everything. Maybe we should seek additional input?

Reply:
This is a general user guide. What is most important to the majority of users/notebooks?

I suggest we just list these:

- ARIA-tools
- cartopy
- curl
- dask
- gdal
- GIAnT
- git
- h5py
- hdf4
- hdf5
- hyp3-lib
- ISCE
- matplotlib
- MintPy
- netcdf4
- numpy
- opencv
- python 3.7.6
- scikit-image
- scipy
- snap
- snaphu
- TRAIN
- vim
- wget
- zip

This covers all the majors that users are likely to want. If they stumble on anything else, more power to them.

Reply:
I left asf-hyp3, python 2.7.17, and python-pyproj in there too. I'll take them out if you feel strongly about it but I think those are packages users will care about.

Reply:
I want to draw people away from Python 2. While a couple packages still require it, I don't want to broadcast it. We can explain how to run them if they ask.

For `asf-hyp3` and `python-pyproj`, I'm indifferent. People who know what `asf-hyp3` is will know what to do with it. I'm not sure if knowing `pyproj` exists is needed but we can leave it in.